ui/components/Layout.js

```diff
@@ -37,17 +42,36 @@ const Layout = ({
         {/* NOTE TO PEOPLE WANTING TO EDIT THIS:
             Please see our license in the file /LICENSE in this repo for details on how you're allowed to change this section */}
         <div>
-          You are using <LinkOut href="https://cobudget.com/">Cobudget</LinkOut>
-          . Source code available{" "}
-          <LinkOut href="https://github.com/cobudget/cobudget">online</LinkOut>.
+          <FormattedMessage
+            defaultMessage="You are using <a1>Cobudget</a1>. Source code available <a2>online</a2>."
+            values={{
+              a1: (msg) => <LinkOut href="https://cobudget.com/">{msg}</LinkOut>,
+              a2: (msg) => <LinkOut href="https://github.com/cobudget/cobudget">{msg}</LinkOut>
+            }}
+          />
         </div>
         <div className="space-x-6">
           {process.env.PRIVACY_POLICY_URL && (
-            <LinkOut href="/privacy-policy">Privacy Policy</LinkOut>
+            <LinkOut href="/privacy-policy">
+              Privacy Policy
+            </LinkOut>
           )}
           {process.env.TERMS_URL && (
             <LinkOut href="/terms-and-conditions">Terms and Conditions</LinkOut>
           )}
+          <select onChange={(e) => changeLocale(e.target.value)}>
```

Comment:
I'm unable to click the dropdown. I think my resolution/zoom makes the page too small

![image](https://user-images.githubusercontent.com/1427063/166241216-77c59d38-c27e-4be9-9438-a549c9e2cef1.png)

Reply:
This probably only applies to the bucket page